ppafm/atomicUtils.py

```diff
@@ -163,12 +166,12 @@ def loadCoefs(characters=["s"]):
     coefs = []
     for char in characters:
         fname = "phi_0000_%s.dat" % char
```

Comment:
I know that this is not connected with this PR, but when are we needing to read `phi_0000_%s.dat` ? I thought, that this for STM.